packages/program-boilerplate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/program-boilerplate",
-  "version": "3.4.1",
+  "version": "3.4.1-alpha.4",
```

Comment:
Shouldn't be merging an alpha version into master, you should deploy a stable version to npm